docs/style-guide.md

```diff
@@ -0,0 +1,46 @@
+# Front-end Style Guide
+
+## Layout
+
+- Mobile-width: 414px
+- Desktop-width: 1440px
+
+```css
+:root {
+  --radius: 15px;
+}
+```
+
+## Colors
+
+**css variable**
+
+```css
+:root {
+  --view-reply-text-color: rgba(61, 166, 255, 1);
+  --video-title-text-color: rgba(255, 255, 255, 1);
+  --video-bar-time-played-bg-color: rgba(255, 179, 71, 1);
+  --video-bar-bg: rgba(255, 255, 255, 0.15);
+  --timeline-text-color: rgba(61, 166, 255, 1);
+  --text-primary-color: rgba(255, 255, 255, 1);
+  --text-input-color: rgba(255, 255, 255, 1);
```

Comment:
repeat color: text-primary-color === text-input-color -> delete: text-input-color

Reply:
it doesn't really repeat, can be tried in the future